admin/src/pages/HomePage/index.js

```diff
@@ -135,66 +200,16 @@ const HomePage = () => {
       });
       if (!collections.length) setModal(true);
     });
-  }, []);
 
+    if (ctx.state.user.role === "Administrator" && ctx.state.isAuthenticated) {
+      setUserPermissions(true) 
+    }
```

Comment:
please check against permission for plugin, something like this:
![image](https://github.com/user-attachments/assets/6ee3dddf-c6df-41d2-836e-1211f0718d8f)